package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "example-full-stack",
+  "version": "1.0.0",
+  "private": false,
+  "description": "XcooBee SDK and Payment SDK usage example",
+  "main": "server.js",
+  "scripts": {
+    "start": "nodemon server.js",
+    "dev": "node server.js"
```

Comment:
This is the first project I've seen that has both a `start` and a `dev` script. In the projects I've looked at, one or the other is used and it is used for development purposes. That is, it is monitoring changes and refreshing the app automatically.

So, I am thinking is might be better to rename `dev` to something like `serve`. Thoughts?